paddle/fluid/operators/detail/send_recv.proto

```diff
@@ -67,6 +67,8 @@ message VariableMessage {
   bytes serialized = 8;
   // selected_rows data
   bytes rows = 9;
+  // prefetch var name
```

Comment:
Look up table block execution output variable name.

Reply:
Done.

Reply:
Seems not updated?

Reply:
Sorry...updated by the comments.